webviz_subsurface/plugins/_seismic_misfit.py

```diff
@@ -0,0 +1,3058 @@
+                ensdf_stat["sim_mean"] - ensdf_stat["sim_p90"]
+            )
+            errory = "error_plus"
+            errory_minus = "error_minus"
+
+        # -------------------------------------------------------------
+        if colorby == "region":
+            ensdf_stat = ensdf_stat.sort_values(by=[colorby])
+            ensdf_stat = ensdf_stat.astype({"region": "string"})
+
+        # -------------------------------------------------------------
+        # get color and zeroline ranges from first case
+        if first:
+            rmax = max(ensdf_stat["sim_mean"].max(), ensdf_stat["obs"].max())
+            rmin = min(ensdf_stat["sim_mean"].min(), ensdf_stat["obs"].min())
+            axis_extend = (rmax - rmin) * 0.10
+            xplot_range = [rmin - axis_extend, rmax + axis_extend]
+            cmin = None
+            cmax = None
+            if colorby is not None:
+                cmin = ensdf_stat[colorby].min()
+                if colorby == "region":
+                    cmax = ensdf_stat[colorby].max()
+                else:
+                    cmax = ensdf_stat[colorby].quantile(0.9)
+            first = False
+
+        dfs.append(ensdf_stat)
+    # -------------------------------------------------------------
+    if len(dfs) == 0:
+        return None
+
+    df_stat = pd.concat(dfs)
+
+    no_plots = len(df_stat.ENSEMBLE.unique())
+    if no_plots <= fig_columns:
+        total_height = figheight * (1 + 45 / figheight)
+    else:
+        total_height = figheight * round(no_plots / fig_columns)
+
+    fig = px.scatter(
+        df_stat,
+        facet_col="ENSEMBLE",
+        facet_col_wrap=fig_columns,
+        # height=total_height,
+        x="obs",
+        y="sim_mean",
+        range_x=plot_range,
+        range_y=plot_range,
+        error_y=errory,
+        error_y_minus=errory_minus,
+        color=colorby,
+        range_color=[cmin, cmax],
+        size=sizeby,
+        size_max=20,
+        hover_data=list(df_stat.columns),
+    )
+    fig.update_traces(marker=dict(sizemode="area"), error_y_thickness=1.0)
+
+    # add zero/diagonal line
+    tracelegend = False
+    # if colorby == "region":
+    #    tracelegend = True
+    fig.add_trace(
+        go.Scattergl(
+            x=xplot_range,
+            y=xplot_range,
+            mode="lines",
+            line_color="gray",
+            name="zeroline",
+            showlegend=tracelegend,
+        ),
+        row="all",
+        col="all",
+        exclude_empty_subplots=True,
+    )
+
+    # marker line color, default is white
+    if sizeby is None:
+        fig.update_traces(
+            marker=dict(line=dict(width=0.4, color="black")),
+            selector=dict(mode="markers"),
+        )
+
+    figures.append(wcc.Graph(figure=fig.to_dict(), style={"height": total_height}))
+    return figures
+
+
+# -------------------------------
+# pylint: disable=too-many-statements, simplifiable-if-expression
```

Comment:
What does the `simplifiable-if-expression` point to? Is it simplifiable, and if not should we move this disable to where it occurs?

Reply:
@anders-kiaer it refers to this line:
`obslegend = True if colorby == "region" else False`

Do you know if this is simplifiable?
If not, would this be ok:
`    # pylint: disable=simplifiable-if-expression
    obslegend = True if colorby == "region" else False
`
I tried to add # pylint at the end of line, but that didn't work

Reply:
Yes, something like this should work 😉 

> obslegend = (colorby == "region")